src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.suppliers;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS =
```

Comment:
be attentive with constant name
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names